sql/internet_outages/italy_covid19_outage_v1/query.sql

```diff
@@ -0,0 +1,192 @@
+DECLARE country_code STRING DEFAULT "IT";
```

Comment:
Given this query is to be run once, I'd argue it should be called `init.sql` rather than `query.sql` and it should explicitly create the destination table via a CREATE TABLE AS query.

BQ scripts are incompatible with destination tables, so this script has to handle putting data where we want it. We can't tell the job to insert the output of the final query into a destination.

Reply:
> 
> 
> > What does that mean for me? Is there anything I need to change here?
> 
> The destination dataset and table name are determined by the directory structure. Could you move `sql/internet_outages/italy_covid19_outage_v1` to `sql/telemetry_derived/italy_covid19_outage_v1`?

Done!